apistructs/k8s.go

```diff
@@ -27,4 +27,5 @@ const (
 type TaskInspect struct {
 	Object runtime.Object    `json:"object"`
```

Comment:
It's not common for all pipeline jobs. It's just for k8s jobs.

Reply:
object filed may change to interface,keep desc filed string